### DISCUSSION TITLE
Amazing work

### DISCUSSION THREAD

**Discussion by blueyred** [gilzoide/unity-lottie-player#1] [Ideas]
Hey Gil, this is amazing, the animations are running so smoothly!
The importer is genius too, really useful to have that level of validation without having to go into play mode.
Small suggestion to help people get started using this; maybe just a few lines about how to set this up in a scene (or add some samples to this)
Maybe all you need is
"add a UI component to a Canvas and then add ImageLottiePlayer"

and maybe add `[AddComponentMenu("UI/ImageLottiePlayer")]` above the ImageLottiePlayer class?

thanks for this repo, super awesome!

**Comment by gilzoide** [comment]
Hey @blueyred, I'm really glad you liked it ^^

> maybe just a few lines about how to set this up in a scene (or add some samples to this)

Yeah, I usually add a "How to use" section in README, sometimes a sample scene, but this package was made somewhat as an experiment so I didn't give it that much love yet.
I was actually trying to manually tesselate the vectors as a mesh, which is quite hard to do efficiently, trying to beat the raster solution rlottie uses and have lots of high definition animations running at once in mobile platforms =P

> maybe add [AddComponentMenu("UI/ImageLottiePlayer")] above the ImageLottiePlayer class?

Sure, why not?

Thank you for the suggestions, Issues #2 and #3 have been created and we should have them addressed at some point.
Cheers \o/
